src/App.js

```diff
@@ -1,7 +1,7 @@
-import './App.css';
```

Comment:
What did you use for the indentation? It doesn't look consistent - we should be using prettier/eslint

Reply:
Will leave this issue for the CI/CD team

Reply:
I think we need a linting script, this should be a separate issue